.gitignore

```diff
@@ -4,3 +4,5 @@
 .idea/modules.xml
 .idea/vcs.xml
 .idea/workspace.xml
+
+.vscode/*
```

Comment:
Why do we need this link?

Reply:
Because my vscode will generate some files under .vscode folder. I dont really want them to be added to the branch, thats why I gitignore that part, so it wont add to the branch when I push the changes.

Reply:
Oh, makes sense! Thanks!